util/cron/test-valgrindexe.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Test valgrind-compatible configuration on full suite with valgrind on linux64.
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common.bash
+source $CWD/common-valgrind.bash
+source $CWD/common-localnode-paratest.bash
+
+# valgrind serializes execution, so don't limit to one executable at a time
+unset CHPL_TEST_LIMIT_RUNNING_EXECUTABLES
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="valgrind"
+
+$CWD/nightly -cron -valgrindexe ${nightly_args} $(get_nightly_paratest_args 20)
```

Comment:
To start I'm on board with keeping `-nodepara 20` here just to see the impact on run times, but if it's fast enough I think we need to drop to 12 or lower.

Reply:
Yeah, that makes sense. We'll watch the times.